victims/models.py

```diff
@@ -27,6 +29,17 @@ def is_valid(self):
 
         return True
 
+    def to_json(self):
+        items = {
+            "url": self.url,
+            "source": self.source,
+            "title": self.title,
+            "image_or_video": self.image_or_video,
+            "summary": self.summary,
+            "case_id": self.case_id,
+        }
```

Comment:
Any special reason to not use `dict(self)` and keep the descriptive verbose approach? IMHO this is technical debt.

Reply:
I used variable here cause I had to somehow serialize `datetime.date`. After my research on network I can propose to implement custom json encoder class, which will handle `datetime.date` value. With usage of `dataclasses.asdict`. Then whole `to_json` method may look like this:

```python
    def to_json(self):
        return json.dumps(asdict(self), cls=DateEncoder, sort_keys=True)
```

Reply:
This is a good way. Alternatively simpler though:

```python
def to_json(self):
    data = asdict(self)
    data['when'] = date['when'].strftime("%Y-%m-%d")
    return json.dumps(data, sort_keys=True)
```